bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/AbstractComponentImpl.java

```diff
@@ -135,7 +137,28 @@ public ComponentData getData() {
         }
         return componentData;
     }
-
+    
+    /**
+     * See {@link Component#getAppliedCssClasses()}
+     *
+     * @return The component styles/css class names
+     */
+    @Override
+    @Nullable
+	public String getAppliedCssClasses() {
+    	
+		ComponentStyleInfo componentStyleInfo = this.resource.adaptTo(ComponentStyleInfo.class);
+		String appliedCssClassNames = null;
+		
+		if(componentStyleInfo != null)
+		{
+			appliedCssClassNames = componentStyleInfo.getAppliedCssClasses();
+		}
+
+		// Returning null so sling model exporters don't return anything for this property if not configured
+		return StringUtils.defaultIfBlank(appliedCssClassNames, null);
```

Comment:
This can be simplified, if not caching the `appliedCssClassNames` then it could just be 

```JAVA
public String getAppliedCssClasses() {
    return Optional.ofNullable(this.resource.adaptTo(ComponentStyleInfo.class))
        .map(ComponentStyleInfo::getAppliedCssClasses)
        .filter(StringUtils::isNotBlank)
        .orElse(null);
}
```

Regardless of if this change is made, can the formatting style remain consistent with the rest of the project, just to make it easier to read?

Reply:
@ky940819 - Taken care of !

Reply:
@ky940819 - This is now taken care of !